0_intro_to_ml_using_jax.ipynb

```diff
@@ -0,0 +1,3758 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.        assert jnp.equal(error, correct_error[i]), f"Incorrect implementation. Value: {error} Expected Value: {correct_error[i]}. Parameters (b,w,x_i,y_i): {b,w,x_i,y_i} "
</pre><p>We might need to put some precision tolarance in here. I got the following error:</p><p><code>Incorrect implementation. Value: 105.4729 Expected Value: 105.47291. Parameters (b,w,x_i,y_i): (3.77, 0.05, 210, 4)</code></p><p> </p><p>Oh, I see why my implementations precision is different. I used base python instead of jnp (we should maybe explicitly say "use jnp to implement...")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='166'/>

Reply:
<p>I think we do mention that jnp.dot is a useful function. Do you think that is enough?</p>